### ISSUE TITLE
Accessibility for Full Stack Developer

### ISSUE THREAD

**Issue by Supravisor** [freeCodeCamp/freeCodeCamp#63610]
### Describe the Issue

Both the sizes for the font and input in the new Full Stack Developer may be too small for accessibility.

Users who interact with the new layout with a touch screen instead of a mouse may experience difficultly selecting lessons, especially the workshops.

Improve the font size and input size, or revert back to the previous version.

For accessibility, the input target area should be 44 px x 44 px
Currently it is 37.8 px x 37.8 px

### Affected Page

https://www.freecodecamp.org/learn/full-stack-developer

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

Keep the layout the same, but make the font and input target area sizes similar to the legacy curriculum.

### Screenshots

<img width="779" height="515" alt="Image" src="https://github.com/user-attachments/assets/e513dd5a-30cf-491c-9664-982db59d1aee" />

<img width="691" height="325" alt="Image" src="https://github.com/user-attachments/assets/4262e5d1-b9ef-4477-9efe-21a7d9e2296b" />

<img width="761" height="570" alt="Image" src="https://github.com/user-attachments/assets/21fb4043-b066-47ba-afb2-123337958131" />

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by huyenltnguyen** [comment]
I think we can bump the square size to 2.6rem x 2.6rem.

Additionally, the gap between squares should be increased. They are currently too close together and could lead to misclicks. The gap is currently 3px, I'd suggest changing it to 6px.
